galoy-client/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "galoy-client"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+futures = "0.3.23"
+graphql_client = {version = "0.11.0", features = ["reqwest-blocking", "reqwest"]}
+reqwest = {version = "0.11.11", features = ["json"]}
+serde = "1.0.144"
+thiserror = "1.0.32"
+tokio = {version = "1.20.1", features = ["full"]}
```

Comment:
why full?

Reply:
Without the feature flag of `full` or `macros`, tokio test was not found. I have set the feature to `macros` so that it compiles. Uncertain why this is the case given that some other crates do not have any features flags.